com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -72,6 +72,21 @@ def transform(self, func):
         op.addInputPort(outputPort=self.oport)
         oport = op.addOutputPort()
         return Stream(self.topology, oport)
+     
+    def multiTransform(self, func):
+        """
+        Transforms each tuple from this stream into 0 or more tuples using the supplied function.
+        For each tuple on this stream, the returned stream will contain all non-None tuples in
```

Comment:
Should the function return an iterable or an iterator?

Or should it be either can be handled, would this approach be more Python-like?